doc/api/stream.md

```diff
@@ -1860,15 +1872,19 @@ It can be overridden by child classes but it **must not** be called directly.
 #### readable.push(chunk[, encoding])
 <!-- YAML
 changes:
+  - version: REPLACEME
+    pr-url: https://github.com/nodejs/node/pull/22427
+    description: The `chunk` argument can now be any `TypedArray` or a
+                 `DataView`.
   - version: v8.0.0
     pr-url: https://github.com/nodejs/node/pull/11608
     description: The `chunk` argument can now be a `Uint8Array` instance.
 -->
 
-* `chunk` {Buffer|Uint8Array|string|null|any} Chunk of data to push into the
-  read queue. For streams not operating in object mode, `chunk` must be a
-  string, `Buffer` or `Uint8Array`. For object mode streams, `chunk` may be
-  any JavaScript value.
+* `chunk` {Buffer|TypedArray|DataView|string|null|any} Chunk of data to push
+  into the read queue. For streams not operating in object mode, `chunk` must
+  be a string, `Buffer` or `Uint8Array`. For object mode streams, `chunk` may
+  be any JavaScript value.
```

Comment:
I guess this sentence is not up-to-date.  It's not just string, `Buffer` or `Uint8Array` anymore.

Reply:
True, but the typed arrays and data views can be passed only in object mode. So thought that the current statement still remains true.

Reply:
Maybe I misunderstood this PR then.  I thought the whole point of this is to support all `TypedArray` and `DataView`  in binary mode. 

This looks like from looking at the code  as `!state.objectMode && Stream._isArrayBufferView` works when `objectMode` is `false.

Reply:
yea... I wasn't sure of the behaviour we required here, which is why I had raised the PR [with a caveat](https://github.com/nodejs/node/pull/22427#issue-209744259).

[The PR](https://github.com/nodejs/node/pull/11608) introducing support for `Uint8Array` in streams, too, [asserts a similar behaviour](https://github.com/nodejs/node/blob/master/test/parallel/test-stream-uint8array.js#L11-L47), so I wasn't sure which behaviour was required here.

Reply:
Exactly. Still, this PR make accepting `TypedArray` and `DataView`  everywhere so it should be reflected in the docs.